client/my-sites/plugins-wpcom/plugin-install-button.jsx

```diff
@@ -49,11 +49,25 @@ export const WpcomPluginInstallButton = props => {
 		}
 	}
 
+	function isDisabled() {
+		// Pressable prevents installation of some plugins, so we need to disable AT for them.
+		// More info here: https://kb.pressable.com/faq/does-pressable-restrict-any-plugins/
+		const unsupportedPlugins = [
+			'nginx-helper',
+			'w3-total-cache',
+			'wp-rocket',
+			'wp-super-cache',
+			'bwp-minify',
+		];
```

Comment:
I was thinking about that too, but thought that it could be worthwhile to leave it in that component too, in case it's reused somewhere else.